Cargo.toml

```diff
@@ -13,6 +17,9 @@ sha2 = "0.8.1"
 serde = "1.0.106"
 bincode = "1.2.1"
 hex = "0.4.2"
+cmdr = { path = "../cmdr/cmdr" }
```

Comment:
what is this cmdr?

Reply:
It's a little lib I found that gives you a nice repl like interface, they hadn't pushed up a release recently hence the path dependency, I've requested they publish to crates.io soon https://github.com/mendelt/cmdr/issues/42

Reply:
ok, we should be able to use a github reference in Cargo.toml meanwhile.  makes it easier to build.

Reply:
done ✔️